frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -196,7 +196,10 @@ export const CurrentVersion: React.SFC<CurrentVersionProps> = ({ cv }) => {
   return lastVersion ? <span className="co-select-to-copy">{lastVersion}</span> : <>None</>;
 };
 
-export const UpdateLink: React.SFC<CurrentVersionProps> = ({ cv }) => {
+export const UpdateLink: React.SFC<CurrentVersionProps> = ({
+  cv,
+  title = 'Update to another version',
```

Comment:
I'd suggest using the same text in both cluster settings and the dashboard instead of making this a prop.

@andybraren @beanh66 thoughts?

Reply:
> <ArrowCircleUpIcon className="update-pending" /> Update available

"Update available" with an up arrow icon? If so then yes, good catch.

Reply:
We can show full update status here via https://github.com/openshift/console/blob/e3080430f4ac297619b57062410c461333e62c9f/frontend/public/components/cluster-settings/cluster-settings.tsx#L162 , means render icons + texts for all update-related states same as on the Cluster Settings page.

If so, should be the `Up to date` msg with the green check icon skipped as an exception for the dashboard? Or should we stay fully consistent?

Reply:
I am sorry, I was wrong - the `UpdateStatus` component would require more customization for the dashboard case.
It renders i.e. `Update now` button which looks ugly on the dashboard.

@andybraren , @spadgett , please confirm we can move forward by following:
New `UpdateStatusDasboard` component will be implemented to render:
- arrow circle up + version as a link leading the user to the update modal when an update is available
- spinner + version as a link leading to the cluster settings page for more details when an update is in progress
- version as a text (non-clickable) in other cases

Reply:
(Sorry for my slow response) If I understand correctly, I'd agree that a new `UpdateStatusDashboard` should be made instead of showing a button to match the Cluster Settings behavior. That would look odd.

Below is what I'm imagining the user would see in each state. The new version string probably would not be shown in this card (it starts looking crowded, maybe it could be included in the Status card's message instead).

Clicking "Update available" reveals the update modal. Clicking "Updating" takes the user to the Cluster Settings page for 4.3.

![card-status-upgrades](https://user-images.githubusercontent.com/9122899/66237321-d00bf800-e6c2-11e9-833c-fc98296f1ac6.png)

Reply:
done, please see screenshots in regular comments bellow